### ISSUE TITLE
Work around for correctly calculating flux from an integrated emission (moment 0 - 2D) image and constraining min_npix?

### ISSUE THREAD

**Issue by AstroJLo** [dendrograms/astrodendro#160]
Hi there,

Firstly, a big thank you to everyone that has worked on updating and maintaining the astrodendro package; i found the tutorial to be very useful and informative at allowing me to understand what dendrograms are and how they can be utilised. Now, onto my questions:

Is it possible to calculate the correct flux from a 2D moment 0 (integrated emission) image of a molecular cloud? The units of the 2D image for `BUNIT` are in K*km/s. I am aware that when i try to do this, i get an error message stating that it isn't possible, due to these units not being implemented yet. However, i am just wondering whether a work around exists for this issue? 

If I use 3D data (i.e. a PPV molecular line cube), this issue doesn't occur and the flux can be correctly calculated when `BUNIT` is equal to 'K' (thanks to the efforts of @tomr-stargazer !). 

The issue with using a data cube (for now) is that it becomes slightly more challenging to choose the correct `min_value`, `min_npix` (this one in particular) and  `min_delta` values; which isn't an issue that occurs with the 2D data. Perhaps @keflavich may have some ideas here for constraining `min_npix` in a data cube? The cloud I'm looking at is 'small' with the x and y dimensions of the image having an area that won't exceed 870" x 870" (29 x 29 pixels, with each pixel increment being 30 arc seconds).

**Comment by keflavich** [comment]
Could you post the traceback you get when using BUNIT = K km/s ?  I don't remember exactly where that is, but it sounds like it might be an equivalency issue that may have been solved.

**Comment by AstroJLo** [comment]
Hi @keflavich , thanks for getting back to me. As per your request, here is a screenshot of the code.

![dendrograms_2d_units_issue_screengrab](https://user-images.githubusercontent.com/36264663/37180768-53e71678-237e-11e8-97e3-bf3fdb7676d8.jpg)

**Comment by keflavich** [comment]
No, I don't have anything.  Could you post a minimum working example?  That might help.

FYI, I'm not an official maintainer of this repository, so I'm not promising anything.
